### PR TITLE
chore(INT-631): update redis expiry time

### PR DESCRIPTION
**Pull request by anantjain45823**
## Description of the change

> Updating redis expiry time to 24 hrs

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -9,3 +9,4 @@ REDIS_PORT = 6379
 REDIS_PASSWORD = 123
 REDIS_USERNAME = abc
 USE_REDIS_DB = true
+redisDefaultExpiryTime = 86400
```
Comment (utsabc) [inline sample.env:12]: This Should be all caps. And default should be 1hr change to 24 hr from helm charts

Comment (anantjain45823) [reply sample.env]: done

**Review by mihir-4116 — DISMISSED** [review]

**Review by aashishmalik — DISMISSED** [review]

File: sample.env [hunk 2]
```diff
@@ -9,3 +9,4 @@ REDIS_PORT = 6379
 REDIS_PASSWORD = 123
 REDIS_USERNAME = abc
 USE_REDIS_DB = true
+REDIS_EXPIRY_TIME = 60*60
```
Comment (utsabc) [inline sample.env:12]:
```suggestion
REDIS_EXPIRY_TIME_IN_SEC = 3600
```

Comment (anantjain45823) [reply sample.env]: Done

**Review by utsabc — APPROVED** [review]

**Review by mihir-4116 — APPROVED** [review]
LGTM

**Pull request merged**